core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/PresentedOfferingContext.kt

```diff
@@ -3,34 +3,36 @@ package com.revenuecat.purchases.kmp
 /**
  * Contains data about the context in which an offering was presented.
  */
-public expect class PresentedOfferingContext
+public expect class PresentedOfferingContext {
```

Comment:
Thanks so much for the explanation. I had leaned the `expect`/`actual` because it worked for pretty much all cases. But if it's discouraged I will change it. I can look into my IDE history, so it shouldn't be too bad to revert